OrbitClientProtos/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ project(OrbitClientProtos)
 
 add_library(OrbitClientProtos STATIC)
 protobuf_generate(TARGET OrbitClientProtos PROTOS capture_data.proto
+                                                  orbit_log_event.proto
```

Comment:
I think that proto should live in `MetricsUploader` instead. It's tied to the class in there, so there is no use case without the class.

Reply:
I've moved it to the `MetricsUploader/proto` folder.